src/workloads/query/TimeSeriesLastpoint.yml

```diff
@@ -95,59 +104,42 @@ LastpointQueryMetaAscTimeAscRepeat100: &LastpointQueryMetaAscTimeAscRepeat100
       Repeat: 100
       Collection: *coll
```

Comment:
[optional] to reduce the runtime of this workload we could consider dropping this to 50 and not 100 since we have way more documents now.